### PR TITLE
feat: new runtimes

### PR DESCRIPTION
**Pull request by loks0n**

### CODE REVIEW

File: src/Runtimes/Runtimes.php [hunk 1]
```diff
@@ -52,6 +55,7 @@ public function __construct($version = '')
         $deno->addVersion('1.21', 'denoland/deno:alpine-1.21.3', 'openruntimes/deno:' . $this->version . '-1.21', [System::X86]);
         $deno->addVersion('1.24', 'denoland/deno:alpine-1.24.3', 'openruntimes/deno:' . $this->version . '-1.24', [System::X86]);
         $deno->addVersion('1.35', 'denoland/deno:alpine-1.35.2', 'openruntimes/deno:' . $this->version . '-1.35', [System::X86]);
+        $deno->addVersion('1.40', 'denoland/deno:alpine-1.40.5', 'openruntimes/deno:' . $this->version . '-1.40', [System::X86, System::ARM64]);
```
Comment (Meldiron) [inline src/Runtimes/Runtimes.php:58]: You added support for arm64 here, on purpose?

Comment (loks0n) [reply src/Runtimes/Runtimes.php:58]: Screenshot 2024-03-01 at 11 28 11

Yup

**Review by Meldiron — APPROVED** [review]

File: src/Runtimes/Runtimes.php [hunk 2]
```diff
@@ -75,16 +80,19 @@ public function __construct($version = '')
...
         $java->addVersion('18.0', 'openjdk/18-jdk-slim', 'openruntimes/java:' . $this->version . '-18.0', [System::X86, System::ARM64, System::ARMV7, System::ARMV8]);
+        $java->addVersion('21.0', 'openjdk/21-jdk-slim', 'openruntimes/java:' . $this->version . '-21.0', [System::X86, System::ARM64, System::ARMV7, System::ARMV8]);
         $this->runtimes['java'] = $java;
 
         $swift = new Runtime('swift', 'Swift', 'src/function/Runtime serve --env production --hostname 0.0.0.0 --port 3000');
         $swift->addVersion('5.5', 'swiftarm/swift:5.5.3-ubuntu-jammy', 'openruntimes/swift:' . $this->version . '-5.5', [System::X86, System::ARM64, System::ARMV7, System::ARMV8]);
         $swift->addVersion('5.8', 'swiftarm/swift:5.8-ubuntu-jammy', 'openruntimes/swift:' . $this->version . '-5.8', [System::X86, System::ARM64, System::ARMV7, System::ARMV8]);
+        $swift->addVersion('5.9', 'swift:5.9-jammy', 'openruntimes/swift:' . $this->version . '-5.9', [System::X86, System::ARM64,  System::ARMV7, System::ARMV8]);
```
Comment (christyjacob4) [inline src/Runtimes/Runtimes.php:89]: why not `ubuntu-jammy` ?

Comment (Meldiron) [reply src/Runtimes/Runtimes.php:89]: I think Luke just copied what he saw in Open RUntimes. swift image as base image should be fine. We do that for most if not all runtimes - we get image that has runtime pre-installed.

Comment (loks0n) [reply src/Runtimes/Runtimes.php:89]: We used `swift:5.9-jammy` as the base image here: https://github.com/open-runtimes/open-runtimes/blob/main/runtimes/swift-5.9/Dockerfile

It should be the same tag as `ubuntu-jammy` but wanted to keep consistent with base image

**Pull request merged**